src/App.js

```diff
@@ -1,22 +1,18 @@
 import logo from './logo.svg';
 import './App.css';
+import Greeting from './components/pure/greeting';
+import GreetingF from './components/pure/greetingF';
+import TaskListComponent from './components/container/task_list';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        {/* <Greeting name={"Miky"}></Greeting> */}
```

Comment:
Try to avoid comments like this on the code, when you are inside the return method.
So we dont render things we dont want.